dandi/dandiapi.py

```diff
@@ -295,6 +296,16 @@ def create_dandiset(self, name: str, metadata: Dict[str, Any]) -> "RemoteDandise
             self, self.post("/dandisets/", json={"name": name, "metadata": metadata})
         )
 
+    def match_schema_version(self, schema_version: Optional[str] = None) -> None:
+        if schema_version is None:
+            schema_version = get_schema_version()
+        server_schema_version = self.get("/info/")["schema_version"]
```

Comment:
you trust dandi-api a lot ;-)  could you make it into something like
```suggestion
        server_info = self.get("/info/")
        server_schema_version = server_info.get("schema_version")
        if not server_schema_version:
                raise RuntimeError(f"Server did not provide schema_version in its /info output: {server_info}")
```
so if api server malfunctions or changes its "API", users would get some informative message which could help us to debug

Reply:
Done.